src/applications/claims-status/tests/components/claim-files-tab/AdditionalEvidencePage.unit.spec.jsx

```diff
@@ -192,145 +192,9 @@ describe('<AdditionalEvidencePage>', () => {
       tree
         .getMountedInstance()
         .UNSAFE_componentWillReceiveProps({ uploadComplete: true });
-      expect(getClaimEVSS.calledWith(1)).to.be.true;
+      expect(getClaim.calledWith(1)).to.be.true;
       expect(router.push.calledWith('your-claims/1/files')).to.be.true;
     });
-
-    // START lighthouse_migration
-    context('cst_use_lighthouse feature toggle', () => {
-      const props = {
-        claim,
-        files: [],
-        params,
-        resetUploads: () => {},
-        router: getRouter(),
-        uploadField: { value: null, dirty: false },
-        filesNeeded: [],
-        filesOptional: [],
-      };
-
-      it('calls getClaimLighthouse when enabled, doesnt show va-alerts', () => {
```

Comment:
should we keep this test since its a light house test and just rename it or does another test cover this? @jerekshoe